### PR TITLE
Issue 2529

### LINKED ISSUE #2529: `webpack serve --help` output showing all negative flags
**Describe the bug**

`webpack serve --help` output showing all negative flags

**What is the current behavior?**
all negative flags are being printed.

**To Reproduce**

`webpack serve --help`

**Expected behavior**
should print these flags in verbose mode only.

**Screenshots**
Screenshot 2021-03-20 at 4 54 13 PM

**Please paste the results of `webpack-cli info` here, and mention other relevant information**

OS - macOS

**Additional context**

### PR DESCRIPTION
**Pull request by alexander-akait**
**What kind of change does this PR introduce?**

bugfix

**Did you add tests for your changes?**

Existing, we will move on snapshots in near future so these problems will be easy to detect

**If relevant, did you update the documentation?**

No need

**Summary**

fixes #2529

**Does this PR introduce a breaking change?**

No

**Other information**

No

### CODE REVIEW

**Review by evenstensberg — APPROVED** [review]
CI is failing though lgtm

**Comment by alexander-akait** [comment]
Yep, need fix tests, we have invalid tests :smile:

**Review by anshumanv — APPROVED** [review]

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -71,9 +71,9 @@ jobs:
           path: |
             node_modules
             */*/node_modules
-          key: webpack-cli-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-${{ hashFiles('**/yarn.lock', './yarn.lock') }}
+          key: b-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-${{ hashFiles('**/yarn.lock', './yarn.lock') }}
```
Comment (snitin315) [inline .github/workflows/nodejs.yml:74]:
```suggestion
          key: ${{ secrets.CACHE_VERSION }}-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-${{ hashFiles('**/yarn.lock', './yarn.lock') }}
```

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -71,9 +71,9 @@ jobs:
...
           restore-keys: |
-            webpack-cli-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-
+            b-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-
```
Comment (snitin315) [inline .github/workflows/nodejs.yml:76]:
```suggestion
            ${{ secrets.CACHE_VERSION }}-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-
```

**Review by snitin315 — COMMENTED** [review]
I think we can add a `CACHE_VERSION` secret in the settings to avoid changing this key again n again if the cache is broken. just go to settings and change value.

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (alexander-akait) [reply .github/workflows/nodejs.yml:74]: I think we need investigate why we don't invalidate cache on windows, I think we need run `lerna bootstrap` after install deps, let's do it in separate PR

Comment (anshumanv) [reply .github/workflows/nodejs.yml:74]: will do

**Pull request merged**